### PR TITLE
README and Favicon

### PR DESCRIPTION
**Pull request by AbhishekkGautam**

### CODE REVIEW

File: favicon_io /site.webmanifest [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```
Comment (nshanbhag20001412) [inline favicon_io /site.webmanifest:1]: I think this can go to .gitignore. Not sure though.

**Review by nshanbhag20001412 — APPROVED** [review]
Rest looks good to me!

**Review by manojkumar8400 — APPROVED** [review]
looks good.

**Review by palash23jain — APPROVED** [review]
good use of CSS variables. You can prefer using rem instead of px even while giving border property. As of now, it is good to go

File: favicon_io /site.webmanifest (hunk 1 above)
Comment (AbhishekkGautam) [reply favicon_io /site.webmanifest:1]: hmm..I'm also not sure

**Pull request merged**